source3/modules/vfs_full_audit.c

```diff
@@ -651,10 +664,10 @@ static int smb_full_audit_connect(vfs_handle_struct *handle,
 
 	pd->success_ops = init_bitmap(
 		pd, lp_parm_string_list(SNUM(handle->conn), "full_audit",
-					"success", NULL));
+					"success", none));
 	pd->failure_ops = init_bitmap(
 		pd, lp_parm_string_list(SNUM(handle->conn), "full_audit",
-					"failure", NULL));
+					"failure", none));
 
```

Comment:
Why are you replacing NULL with "none" ?

Reply:
Cause NULL is equivalent to "all", which we exactly trying to avoid here. And `none` is the name of the corresponding structure, that makes "none" a default value for the option.